popx/migrator.go

```diff
@@ -431,8 +431,8 @@ func (m MigrationStatuses) HasPending() bool {
 	return false
 }
 
-func (m *Migrator) migrationTableName(ctx context.Context, con *pop.Connection) string {
-	return con.MigrationTableName()
+func (m *Migrator) sanitizedMigrationTableName(con *pop.Connection) string {
+	return regexp.MustCompile(`\W`).ReplaceAllString(con.MigrationTableName(), "")
```

Comment:
We could for performance reasons make the regexp a global, but I guess we don't use it during "normal" operation, but only when applying/checking migrations so it is not really of great concern.